packages/mobx-undecorate/cli.js

```diff
@@ -3,7 +3,7 @@ const { spawn } = require("child_process")
 
 // this is pretty lame, probably better make a .cmd and .sh file...
 spawn(
-    "node_modules/.bin/jscodeshift",
+    "node_modules\\.bin\\jscodeshift",
```

Comment:
this will break on any unix system. Could you use path.separator or path.join instead?